samples/snippets/fsharp/azure/file-storage.fsx

```diff
@@ -12,11 +12,6 @@ open Azure.Storage.Files.Shares.Models // Namespace for ShareServiceProperties
 //
 
 let storageConnString = "..." // fill this in from your storage account
-(*
-// Parse the connection string and return a reference to the storage account.
-let storageConnString = 
-    CloudConfigurationManager.GetSetting("StorageConnectionString")
-*)
 
 //
 // Create the File service client.
```

Comment:
My previous comment should not have been resolved.   We're still not disposing properly.   For example, the streams on L43 and L54.